components/SpotDetails/SpotinfoDesktop.js

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+import Image from "next/image";
+import toFavorite from "../../public/images/toFavorite.svg";
+import onFavorite from "../../public/images/onFavorite.svg";
+import share from "../../public/images/share.svg";
+import navigateTo from "../../public/images/navigateTo.svg";
+import Link from "next/link";
+
+export default function SpotInfoDesktop({
+  image,
+  name,
+  description,
+  winddirection,
+  surfcenter,
+  parking,
+  camping,
+  lat,
+  lon,
+  id,
+  favoriteSpots,
+  setFavoriteSpots,
+}) {
+  const link = `https://www.google.com/maps/search/?api=1&query=${lat}%2C${lon}`;
+
+  function addFavorite() {
+    setFavoriteSpots([...favoriteSpots, id]);
+  }
+
+  function deleteFavorite() {
+    setFavoriteSpots(favoriteSpots.filter((spot) => spot !== id));
+  }
+
+  const isFavOrNot = favoriteSpots.includes(id);
+
+  async function handleShare() {
+    if (navigator.share) {
+      try {
+        const shareData = {
+          title: "Spotifinder",
+          text: "Entdecke diesen Surfspot",
+          url: location.href,
+        };
+        return await navigator.share(shareData);
+      } catch {
+        console.error("failed");
+      }
+    } else {
+      navigator.clipboard.writeText(location.href);
+      alert("In zwischenablage kopiert");
+    }
+  }
+
+  return (
+    <>
+      <StyledSection>
+        <StyledContainer>
+          <StyledImage
+            src={image}
+            alt={`cover image of ${name}`}
```

Comment:
same here, the alt tag should not contain the word image or picture. The name here would be sufficient.